.github/workflows/test-application.yaml

```diff
@@ -13,37 +13,28 @@ jobs:
         runs-on: ubuntu-latest
 
         env:
-            SYMFONY__DATABASE__DRIVER: pdo_mysql
-            SYMFONY__DATABASE__VERSION: 5.7
-            SYMFONY__DATABASE__HOST: 127.0.0.1
-            SYMFONY__DATABASE__PORT: 3306
-            SYMFONY__DATABASE__NAME: sulu_test
-            SYMFONY__DATABASE__USER: root
-            SYMFONY__DATABASE__PASSWORD: root
+            DATABASE_URL: 'mysql://root@127.0.0.1/sulu_redirect_test?serverVersion=5.7'
+            SYMFONY_DEPRECATIONS_HELPER: weak
 
         strategy:
             fail-fast: false
             matrix:
                 include:
-                    - php-version: '5.5'
-                      composer-flags: '--prefer-dist --no-interaction'
-                      tools: 'composer:v1'
+                    - php-version: '7.2'
                       lint: false
-                      env:
-                          SYMFONY_DEPRECATIONS_HELPER: disabled
+                      composer-flags: '--prefer-lowest --prefer-dist --no-interaction'
+                      tools: 'composer:v1'
 
-                    - php-version: '7.2'
+                    - php-version: '7.4'
+                      lint: true
                       composer-flags: '--prefer-dist --no-interaction'
                       tools: 'composer:v2'
-                      lint: true
-                      env:
-                          SYMFONY_DEPRECATIONS_HELPER: weak
 
         services:
             mysql:
                 image: mysql:5.7
                 env:
-                    MYSQL_ROOT_PASSWORD: root
+                    MYSQL_ALLOW_EMPTY_PASSWORD: true
```

Comment:
empty password is okay for me, but for consistency we have in all other bundles the password `root` 🤔

Reply:
We only started using `MYSQL_ROOT_PASSWORD: root` in bundles where jackrabbit is used, because jackrabbit somehow needs that. But yes, I can change it for consistency

Reply:
Ok than it was my bad and we introduced in in the redirect / form / theme bundle without jackrabbit 🙈